package.json

```diff
@@ -3,14 +3,17 @@
     "private": true,
     "type": "module",
     "scripts": {
-        "dev": "concurrently 'pnpm run watch:content' 'pnpm run dev:docs'",
-        "dev:docs": "pnpm --stream --filter {docs} dev:docs",
-        "watch:content": "chokidar 'packages/components/**/*.mdx' -c 'pnpm run dev:content'",
-        "dev:content": "pnpm --stream --filter {docs} dev:contentlayer",
+        "storybook": "concurrently 'pnpm run storybook:new' 'pnpm run storybook:legacy' 'pnpm run storybook:main'",
```

Comment:
```suggestion
        "storybook": "concurrently 'pnpm -r storybook:watch' 'pnpm run storybook:main'",
```

Reply:
i kept it as a separate script and used --filter {packages/*} to only executed it in the packages folder and not the storybook-docs package

Reply:
script was adjusted